Emby.Server.Implementations/Library/IgnorePatterns.cs

```diff
@@ -64,10 +76,13 @@ public static class IgnorePatterns
             "**/.grab/**",
             "**/.grab",
 
-            // Unix hidden files and directories
-            "**/.*/**",
+            // Unix hidden files
             "**/.*",
```

Comment:
```suggestion
            "**/.*[!/]",
```

And add this test: `[InlineData("/media/music/Big K.R.I.T.", false)]` 
to fix #3635

Reply:
Well, that's annoying.
This would break dotfile matching.

Reply:
I fail to see how that's getting matched in the first place… It should only match paths who's filename starts with a dot…

Reply:
> I fail to see how that's getting matched in the first place… It should only match paths who's filename starts with a dot…

That was my thought, but I brought the matching code out to a test project and that's the pattern that matched it

Reply:
I don't think we currently can fix this short of changing to regex or a different glob library.
I submitted an upstream issue; https://github.com/dazinator/DotNet.Glob/issues/78